benchmarks/scripts/run_vera_benchmarks.py

```
@@ -544,12 +573,13 @@ def verify_assertions(vera_results: dict[str, dict]) -> list[tuple[str, bool, st
-    checks.append((
-        "p95 query latency < 500ms on 20+ queries (local computation)",
-        bm25_p95 < 500 and bm25_queries >= 17,
-        f"bm25_p95={bm25_p95:.1f}ms ({bm25_queries} queries), "
-        f"hybrid_p95={hybrid_p95:.1f}ms ({hybrid_queries} queries, includes API round trips)",
-    ))
+    if bm25_queries >= 17:
+        checks.append((
+            "p95 query latency < 500ms on 20+ queries (local computation)",
+            bm25_p95 < 500,
+            f"bm25_p95={bm25_p95:.1f}ms ({bm25_queries} queries), "
+            f"hybrid_p95={hybrid_p95:.1f}ms ({hybrid_queries} queries, includes API round trips)",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)